Pod/Classes/VOKCoreDataManager.m

```diff
@@ -93,7 +95,7 @@ - (NSManagedObjectContext *)tempManagedObjectContext
         return nil;
     }
 
-    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSConfinementConcurrencyType];
+    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
```

Comment:
In any case, this change should be documented in the README or the header comments.